bindings/go/src/fdb/tuple/tuple.go

```diff
@@ -72,6 +73,37 @@ type Tuple []TupleElement
 // an instance of this type.
 type UUID [16]byte
 
+// Versionstamp .
+type Versionstamp struct {
+	TransactionVersion [10]byte
+	UserVersion        uint16
+}
+
+var incompleteTransactionVersion = [10]byte{0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF}
+
+const versionstampLength = 13
```

Comment:
`versionstampLength` is not an accurate name. I should just change that to be `12` and at the usage site use `versionstampLength + 1` instead.